org.eclipse.wildwebdeveloper/build.properties

```diff
@@ -4,6 +4,7 @@ bin.includes = META-INF/,\
                .,\
                plugin.xml,\
                language-servers/node_modules/,\
+               language-servers/,\
```

Comment:
One issue here is that by importing the whole folder, it also adds in the jar the .tgz files that are used at build time and which aren't desired.
@yanxi0830 Please try to remediate that and make only the xml-ls gets added. Feel free to put it in a subfolder if this helps.